crates/uv/tests/venv.rs

```diff
@@ -29,11 +29,8 @@ fn create_venv() -> Result<()> {
             r"Using Python 3\.\d+\.\d+ interpreter at: .+",
             "Using Python [VERSION] interpreter at: [PATH]",
         ),
-        (&filter_venv, "/home/ferris/project/.venv"),
-        (
-            filter_prompt,
-            "Activate with: source /home/ferris/project/.venv/bin/activate",
-        ),
+        (filter_prompt, "Activate with: source .venv/bin/activate"),
+        (&filter_venv, ".venv"),
```

Comment:
Do we still need this filter?

Reply:
Yeah, for some reason on macOS these paths are sometimes prefixed with `/private/`